### ISSUE TITLE
Microweber hanged during installation

### ISSUE THREAD

**Issue by sanyaade-teachings** [microweber/microweber#744]
Hi Everyone,

Many thanks for creating this awesome rapid website development tool.

I was trying to setup microweber for my daughter but the installation hangs at  57%  at installing White label WHMCS module.
After refresh and restarting the installation, it did not go past that module. It hangs at same 57%. installing White label WHMCS

![image](https://user-images.githubusercontent.com/4993211/118265875-b5f6e600-b4b1-11eb-9675-6b657eb14637.png)

Please help me!

God blesses!!!

Best regards,
Sanyaade

**Comment by peter-mw** [comment]
hi, do you see any error ? 

can you check in folder `storage/logs/`

**Comment by sanyaade-teachings** [comment]
@peter-mw :
Many thanks for your support and quick response.

No there was no error-log in the storage/logs/  folder is empty apart from .gtignore which is also an empty file
![image](https://user-images.githubusercontent.com/4993211/118357493-5b718e80-b572-11eb-9511-4d5854f93d05.png)

My Server is running Centos 7 server.
Once again, many thanks for your support.

Hear from you soon!

God blesses!!!

Best regards,
Sanyaade

**Comment by peter-mw** [comment]
Hello, can you download the zip again from here  https://microweber.org/download   and try again

**Comment by sanyaade-teachings** [comment]
@peter-mw : 
 
Just follow your link above, I downloaded the zip edition, extracted it, perform the install steps but it still hangs on installing White label modules. It looks an issue with that module:
 
![image](https://user-images.githubusercontent.com/4993211/118363851-2f640680-b58e-11eb-9ea7-f581d69237fc.png)

God blesses!!!

Best regards,
Sanyaade

**Comment by sanyaade-teachings** [comment]
@peter-mw : 

Many thanks for your continuous support. I have finally got microweber installed on my server with the following steps:

1.) Download and installed php composer on my server. I did the global installation so that I can run it from anywhere on my server:  Plese see below for what I did : 

- hp -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"

- php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"

- php composer-setup.php

See here for more information on the above: https://getcomposer.org/download/ 

Next I move composer.phar to my system bin so that I can run it globally:
         mv composer.phar /usr/local/bin/composer

2.) I clone microweber from its Git repository:
     $ git clone https://github.com/microweber/microweber.git

then cd into it and run the install command:
    cd microweber
    composer install

3.) Next point my web browser to the folder: localhost/mysite/microweber. I follow the microweber install screen but I create a database for my MYSQL (I did not use SQLite). Fill-in all the parameters and click the install button to have microweber install and running as shown below:
![image](https://user-images.githubusercontent.com/4993211/118370024-af459d00-b59d-11eb-9be9-fba372b0cd4b.png)

Many thanks for your support.
 
My next steps is to learn microweber further, see how to adapt it for my daughter web experience and use it in my teachings. My PHP skills is still a bit rusticated but let see how far I can go :)

Once again, many thanks for your continuous support. Much appreciated!

God blesses!!!

Best regards,
Sanyaade
